### PR TITLE
Improve the performance a little.

### PR DESCRIPTION
**Pull request by wlzla000**
Made use of the existing (declared) variable `LEADER` instead of re-obtaining `ALPHABET[0]`, so as to improve the performance a little.
And used brackets instead of `.charAt` in order to give it consistency in its code style.

### CODE REVIEW

**Comment by fanatid** [comment]
Not sure about `charAt`/`[]`.
`charAt(0)` => `LEADER` looks good.

File: index.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ module.exports = function base (ALPHABET) {
     var string = ''
 
     // deal with leading zeros
-    for (var k = 0; source[k] === 0 && k < source.length - 1; ++k) string += ALPHABET[0]
+    for (var k = 0; source[k] === 0 && k < source.length - 1; ++k) string += LEADER
```
Comment (dcousens) [inline index.js]: ACK,  but ditch the other changes

**Comment by dcousens** [comment]
Agreed with @fanatid

**Comment by fanatid** [comment]
@wlzla000 make squash please

**Comment by wlzla000** [comment]
It was my first time squashing pushed commits that I struggled to do it. I'm sorry. Squashing done.

**Comment by fanatid** [comment]
Thank you!

**Pull request merged**